mito-ai/src/Extensions/ErrorMimeRenderer/ErrorMimeRendererPlugin.tsx

```diff
@@ -64,14 +64,34 @@ class AugmentedStderrRenderer extends Widget implements IRenderMime.IRenderer {
     /**
      * Render the original error message and append the custom prompt.
      */
-    async renderModel(model: IRenderMime.IMimeModel): Promise<void> {    
+    async renderModel(model: IRenderMime.IMimeModel): Promise<void> {
+        // Determine if it's an error or a warning
+        const isErrorMessage = 'application/vnd.jupyter.error' in model.data;
+
+        // Create the container for the custom UI elements
         const resolveInChatDiv = document.createElement('div');
-        ReactDOM.render(<ErrorMessage onDebugClick={() => this.openChatInterfaceWithError(model)} />, resolveInChatDiv);
-        this.node.appendChild(resolveInChatDiv);
 
-        // Get the original renderer and append it to the output
+        // Only show the Fix Error in AI Chat button if it is an error, not a warning
+        if (isErrorMessage) {
+            createRoot(resolveInChatDiv).render(
+                <ErrorMessage onDebugClick={() => this.openChatInterfaceWithError(model)} />
+            );
+        }
+
+        // Append the chat container before rendering the original output
+        this.node.appendChild(resolveInChatDiv);
+        
+        // Render the original content
         await this.originalRenderer.renderModel(model);
-        this.node.appendChild(this.originalRenderer.node);
+        const originalNode = this.originalRenderer.node;
+
+        // Apply styling for warnings
+        if (!isErrorMessage) {
+            originalNode.style.background = 'var(--jp-warn-color3)';
```

Comment:
This looks awesome on light mode, but the text becomes hard to read in dark mode. 

Is there an easy fix for this? Now that we're touching more of the Jupyter UI, I want to do a better job of supporting dark mode as well. 

<img width="1026" alt="Screenshot 2024-12-20 at 10 33 36 AM" src="https://github.com/user-attachments/assets/9e2a1ca8-ec05-48a4-ab46-57ea3f454637" />

Reply:
_JupyterLab Dark_
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/5ea74e8d-ced0-4416-af10-f2db950e20d3" />
_JupyterLab Dark High Contrast_
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/ed926e9d-e036-4445-8e42-60443b710310" />
_JupyterLab Light_
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/c9d9ae85-3a95-474c-ad20-1e89b003b84c" />